os/dbuild.sh

```diff
@@ -247,6 +250,103 @@ function SELECT_CONFIG()
 	CONFIGURE ${BOARD}/${CONFIG} || exit 1
 }
 
+function SELECT_PORT()
+{
+	# Get current target board information
+	if [ -z ${BOARD} ]; then
+		SELECT_BOARD
+	fi
+
+	case ${BOARD} in
+	artik053|artik053s|artik055s)
+		unset SELECTED_PORT
+		CFG_FILE=${CONFIGDIR}/artik05x/scripts/artik05x-usb-ftdi.cfg
```

Comment:
This path can be changed when file name or path is changed.
How about moving this to the above?